src/Nethermind/Nethermind.Core/Extensions/IListExtensions.cs

```diff
@@ -14,12 +14,78 @@
 //  You should have received a copy of the GNU Lesser General Public License
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 
+using System;
 using System.Collections.Generic;
 
 namespace Nethermind.Core.Extensions
 {
     public static class IListExtensions
     {
         public static T GetItemRoundRobin<T>(this IList<T> array, long index) => array.Count == 0 ? default : array[(int) (index % array.Count)];
```

Comment:
I suggest only moving things to Nethermind.Core when they are used in at elast 2 projects or cause some dependencues issues -I keep pruning the classes that grow here